indico/web/forms/fields.py

```diff
@@ -374,6 +375,21 @@ def _get_data(self):
         return [] if self.data is None else [self.data]
 
 
+class PersonLinkListFieldBase(EventPersonListField):
+
+    #: Class representing the PersonLink
+    person_link_cls = None
+
+    #: widget
```

Comment:
Better don't put a docstring at all. This one is not useful at all and just adds noise ;)